composer.json

```diff
@@ -14,12 +14,12 @@
         }
     ],
     "require": {
-        "php": ">=5.3"
+        "php": "^7.4"
     },
     "require-dev": {
-        "php": ">=5.6",
-        "phpunit/phpunit": "^5.0",
-        "satooshi/php-coveralls": "^1.0"
+        "friendsofphp/php-cs-fixer": "^2.17",
+        "phpstan/phpstan": "^0.12",
+        "phpunit/phpunit": "^9.5"
```

Comment:
`^8.5.13||^9.5.0` should work fine to keep 7.2 support, without additiona changes...